### ISSUE TITLE
LightningSalesConsole feature doesn't provide full feature

### ISSUE THREAD

**Issue by bingincap** [forcedotcom/cli#1176]
### Summary

The feature name LightningSalesConsole used in scratch org definition files doesn't seem to enable the full feature.

### Steps To Reproduce:

**Repository to reproduce:** [dev-group-feb-2021](https://github.com/bingwang0/dev-group-feb-2021)

1.  Use the definition file config/more-features-scratch.json to create a scratch org
2.  Open the scratch org

### Expected result

Sales Console permission set license is available to use in the scratch org, and there should be a Salesforce Console User permission set (name: force__SalesConsoleUser) available too.

### Actual result

1.  The Sales Console license is available but shows a "disabled" status
2.  There's no expected permission set
The Sales Console Lightning app is avaialble.

### System Information

{
        "cliVersion": "sfdx-cli/7.113.0",
        "architecture": "darwin-x64",
        "nodeVersion": "node-v14.17.4",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.2.2 (core)",
                "@oclif/plugin-not-found 1.2.4 (core)",
                "@oclif/plugin-plugins 1.10.1 (core)",
                "@oclif/plugin-update 1.4.0-3 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.0 (core)",
                "@oclif/plugin-which 1.0.3 (core)",
                "@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "@salesforce/sfdx-trust 3.6.0 (core)",
                "alias 1.1.10 (core)",
                "apex 0.2.3 (core)",
                "auth 1.7.1 (core)",
                "config 1.2.23 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.0 (core)",
                "generator 1.1.7 (core)",
                "limits 1.2.1 (core)",
                "org 1.6.9 (core)",
                "salesforce-alm 52.2.4 (core)",
                "schema 1.0.8 (core)",
                "sfdx-cli 7.113.0 (core)",
                "source 1.0.7 (core)",
                "telemetry 1.2.3 (core)",
                "templates 52.1.0 (core)",
                "user 1.4.0 (core)"
        ],
        "osVersion": "Darwin 20.6.0"
}

### Additional information

Tested to behave the same with a different Dev Hub with CLI running on Windows.

{
        "cliVersion": "sfdx-cli/7.113.0",
        "architecture": "win32-x64",
        "nodeVersion": "node-v14.17.4",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.2.2 (core)",
                "@oclif/plugin-not-found 1.2.4 (core)",
                "@oclif/plugin-plugins 1.10.1 (core)",
                "@oclif/plugin-update 1.4.0-3 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.0 (core)",
                "@oclif/plugin-which 1.0.3 (core)",
                "@Salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "@Salesforce/sfdx-trust 3.6.0 (core)",
                "alias 1.1.10 (core)",
                "apex 0.2.3 (core)",
                "auth 1.7.1 (core)",
                "config 1.2.23 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.0 (core)",
                "generator 1.1.7 (core)",
                "limits 1.2.1 (core)",
                "org 1.6.9 (core)",
                "salesforce-alm 52.2.4 (core)",
                "schema 1.0.8 (core)",
                "sfdx-cli 7.113.0 (core)",
                "source 1.0.7 (core)",
                "telemetry 1.2.3 (core)",
                "templates 52.1.0 (core)",
                "user 1.4.0 (core)"
        ],
        "osVersion": "Windows_NT 10.0.18363"
}

**Comment by mshanemc** [comment]
https://help.salesforce.com/s/articleView?id=sf.console_lex_assign_perm_set.htm&type=5

> NOTE Users with Salesforce licenses, such as Sales Cloud and Service Cloud feature licenses, already have the Lightning Console User permission enabled. You can’t disable this user permission for these users. Don’t assign the Lightning Console permission set license to those with Salesforce user licenses. These users can already access Lightning console apps.

Do you create other license types (Platform?) in your scratch org that you're trying to assign the PSL to?  The setup guide (linked above) seems to suggest that you'd create the PS yourself?

**Comment by bingincap** [comment]
Yes the scratch def does have the Platform User feature in there ("ForceComPlatform").  My main issue here is that the existing repo I'm working with has several Permission Set Groups that utilizes a permission set ("force__SalesConsoleUser") that seems to come with that particular PSL.  I understand that my workaround here for those could be to just clone that PS into a custom one, but rather not do that if there's bug to fix here.  Besides, that wouldn't help if I actually need to assign the PSL to a Platform user with the PSL showing as "disabled".

**Comment by mshanemc** [comment]
marking this as a bug so it ends up in GUS, found the right team to get it resolved.
